include/constants/battle.h

```diff
@@ -122,6 +122,8 @@
 #define STATUS1_PSN_ANY          (STATUS1_POISON | STATUS1_TOXIC_POISON)
 #define STATUS1_ANY              (STATUS1_SLEEP | STATUS1_POISON | STATUS1_BURN | STATUS1_FREEZE | STATUS1_PARALYSIS | STATUS1_TOXIC_POISON | STATUS1_FROSTBITE)
 
+#define STATUS1_REFRESH          (STATUS1_POISON | STATUS1_BURN | STATUS1_PARALYSIS | STATUS1_TOXIC_POISON | STATUS1_FROSTBITE)
```

Comment:
Refresh doesn't cure Freeze, do we classify Frostbite as being equivalent to burn handling or freeze handling?

Reply:
Ye, I think we've decided on this when the pr was made because it makes sense that frostbite follows from burn.

Reply:
Sounds good! It's gonna need an extra line in one of the tests, sec